test/runner/requirements/integration.txt

```diff
@@ -6,4 +6,5 @@ paramiko
 passlib
 pexpect
 pycrypto
+pyOpenSSL
```

Comment:
Install this using a task at the start of the test. That way it won't be required for tests which do not use it.